src/NServiceBus.AcceptanceTests/Routing/MessageDrivenSubscriptions/When_unsubscribing_to_scaled_out_publisher.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing.MessageDrivenSubscriptions
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_unsubscribing_to_scaled_out_publisher : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public Task Should_send_unsubscription_message_to_each_instance()
+        {
+            return Scenario.Define<Context>()
+                .WithEndpoint<ScaledOutPublisher>(b => b.CustomConfig(c => c.MakeInstanceUniquelyAddressable("1")))
+                .WithEndpoint<ScaledOutPublisher>(b => b.CustomConfig(c => c.MakeInstanceUniquelyAddressable("2")))
+                .WithEndpoint<Unsubscriber>(b => b.When(s => s.Unsubscribe<MyEvent>()))
+                .Done(c => c.PublisherReceivedUnsubscription.Count >= 2)
+                .Repeat(r => r.For<AllTransportsWithMessageDrivenPubSub>())
+                .Should(c =>
+                {
+                    // each instance should receive an unsubscription message
+                    Assert.That(c.PublisherReceivedUnsubscription, Does.Contain("1"));
+                    Assert.That(c.PublisherReceivedUnsubscription, Does.Contain("2"));
+                    Assert.That(c.PublisherReceivedUnsubscription.Count, Is.EqualTo(2));
+                })
+                .Run();
+        }
+
+        class Context : ScenarioContext
+        {
+            public List<string> PublisherReceivedUnsubscription { get; } = new List<string>();
+        }
+
+        class ScaledOutPublisher : EndpointConfigurationBuilder
+        {
+            public ScaledOutPublisher()
+            {
+                // store the instance discriminator of each instance receiving a unsubscription message:
```

Comment:
Same typo (at least according to http://english.stackexchange.com/questions/224293/is-it-correct-to-say-unsubscription )